plugins/kubernetes/README.md

```diff
@@ -67,7 +67,8 @@ environment variables, secrets, etc and then sends them to the Kubernetes API.
 Multiple roles will likely be similar, but have different commands or liveness probes.
 ([kucodiff](https://github.com/grosser/kucodiff) can be used to make sure they stay in sync).
 
-Validate required environment variables are set by adding `metadata.annotations.samson/required_env` (space separated).
+Validate required environment variables are set by adding `metadata.annotations.samson/required_env` (space separated) 
+to any template (what includes `spec.containers`).
```

Comment:
I would have said.. to your pod definition. (For example, for a deployment `spec.template.manifest.annotations.samson/required_env`)

Reply:
the any template sounds a little odd